inputfiles/patches/file-api.kdl

```diff
@@ -0,0 +1,10 @@
+interface FileReader {
+  event loadstart type=ProgressEvent
+  event progress type=ProgressEvent
+  event load type=ProgressEvent
+  event abort type=ProgressEvent
+  event error type=ProgressEvent
+  event loadend type=ProgressEvent
```

Comment:
I have a mixed feeling of event types not being in events.kdl, but I guess this is fine.